apps/web/src/components/FormCard.tsx

```diff
@@ -54,21 +54,20 @@ export const FormCard = ({
                     label={
                       <>
                         <Text fontSize="16px" fontWeight="bold">
-                          {signature.signerPosition.name}
+                          {signature.assignedUser?.firstName +
```

Comment:
Should we do a conditional check to see if it exists? And in the else case, say none.